ci-pr.nix

```diff
@@ -34,7 +34,7 @@ let
           fi
         done
 
-        if ! find -name \*.wasm | grep -q .
+        if ! ls */*.wasm
```

Comment:
Hmm, I dislike using non- matching globs like this. And using LS should not be needed: the check is already “done” when the bash is done expanding the glob, the `ls` kinda does unnecessary extra stuff.

It's surprising that this is relatively non-trivial to do elegantly…

Reply:
How about this? https://stackoverflow.com/a/34195247

Reply:
compgen isn't available in the CI env

Reply:
@nomeata this ok?